### ISSUE TITLE
如何在Electron+Vue3+Vite中使用？

### ISSUE THREAD

**Issue by classfang** [bazingaedward/monaco-editor-vue3#24]
按照README中Vite的Demo进行配置后，未成功加载出编辑器，控制台没报错。

**Comment by klavsbuss** [comment]
Yes. Tried to debug, and it seems that its not properly initialising component :(
Seems to be failing here:
<img width="733" alt="image" src="https://github.com/user-attachments/assets/930a141e-f9a1-45fb-8aa7-8b3c8a29568e">
